### PR TITLE
Session 04 coding assignment

### PR DESCRIPTION
**Pull request by iswarya15**

### CODE REVIEW

File: session 04/src/Header.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import TeamLogo from "./assets/react.png";
+
+export default Header = () => {
```
Comment (nitishnivedan) [inline session 04/src/Header.js]: Header can be returned without return keyword.
e.g. export default Header = ()=> (<div>.........</div>)

File: session 04/src/data.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+export const teammates = [
```
Comment (nitishnivedan) [inline session 04/src/data.js:1]: constants in a new file altogether. nice.

File: session 04/src/style.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
```
Comment (shaksham08) [inline session 04/src/style.css:1]: Great Font

**Review by shaksham08 — APPROVED** [review]
Looks good to me

File: session 04/src/style.css (hunk 3 above)
Comment (iswarya15) [reply session 04/src/style.css:1]: Thanks!

File: session 04/src/Header.js (hunk 1 above)
Comment (iswarya15) [reply session 04/src/Header.js]: Removed.

**Pull request merged**